From d8259afbfcf9f6a51cd1c8560ba554e4e298f4ef Mon Sep 17 00:00:00 2001
From: Sveinung Kvilhaugsvik <sveinung84@users.sourceforge.net>
Date: Fri, 20 Apr 2018 21:55:01 +0200
Subject: [PATCH] Properly break "Attack" unit help generation.

The custom action help generated for the "Attack" action in the unit help
would only break if game.info.tired_attack was set. Move the break to the
correct location.

See hrm Bug #748407
---
 client/helpdata.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/helpdata.c b/client/helpdata.c
index 469d0fb8d7..9fe9a8c784 100644
--- a/client/helpdata.c
+++ b/client/helpdata.c
@@ -2556,8 +2556,8 @@ char *helptext_unit(char *buf, size_t bufsz, struct player *pplayer,
                        _("  * weaker when tired. If performed with less "
                          "than a single move point left the attack power "
                          "is reduced accordingly.\n"));
-          break;
         }
+        break;
       case ACTION_CONVERT:
         cat_snprintf(buf, bufsz,
                      /* TRANS: %s is a unit type. "MP" = movement points. */
-- 
2.11.0

